enigmahandlers/sensedialer.go

```diff
@@ -69,13 +70,13 @@ func setupDialer(dialer *enigma.Dialer, timeout time.Duration) {
 				}
 			}
 			if len(cookieStrings) > 0 {
-				httpHeader.Add("Cookie", strings.Join(cookieStrings, "; "))
+				wsHeader.Add("Cookie", strings.Join(cookieStrings, "; "))
```

Comment:
So we want to add the cookies only for the dialer but not keep them further?

Reply:
They are usually in the cookiejar, which is supported by most other things, this websocket implementation however doesn't support cookiejar so need to add them to the rest of the headers manually. But yes, the cookie header should only be added here.